crates/intrinsic-test/src/common/argument.rs

```diff
@@ -168,22 +166,18 @@ where
     /// ARM-specific
     pub fn load_values_c(&self, indentation: Indentation) -> String {
         self.iter()
-            .filter_map(|arg| {
-                // The ACLE doesn't support 64-bit polynomial loads on Armv7
-                // This and the cast are a workaround for this
-
-                (!arg.has_constraint()).then(|| {
-                    format!(
-                        "{indentation}{ty} {name} = cast<{ty}>({load}(&{name}_vals[i]));\n",
-                        ty = arg.to_c_type(),
-                        name = arg.name,
-                        load = if arg.is_simd() {
-                            arg.ty.get_load_function(Language::C)
-                        } else {
-                            "*".to_string()
-                        }
-                    )
-                })
+            .filter(|&arg| (!arg.has_constraint()))
```

Comment:
```suggestion
            .filter(|&arg| !arg.has_constraint())
```

Reply:
I'll fix that, including two more occurrences of `.filter(|&arg| (!arg.has_constraint()))`.